variables.tf

```diff
@@ -132,7 +132,7 @@ variable "management_endpoint_type_for_bucket" {
 variable "retention_enabled" {
   description = "Retention enabled for COS bucket. Only used if 'create_cos_bucket' is true."
   type        = bool
-  default     = true
+  default     = false # retention disabled for test environments - enable for stage/prod
```

Comment:
This would be a breaking change. Everybody relying on the default value of true WILL see regressions as terraform tries to tear down and recreate the bucket with a different retention. Further the tear down will probably fail, since nobody can delete a bucket containing objects with retention policies (until those policies are expired).